packages/payment-proxy-client/src/App.tsx

```diff
@@ -59,11 +59,9 @@ function Copyright(props: any) {
 }
 
 function computePercentagePaid(info: PaymentChannelInfo): number {
-  return (
-    Number(
-      info.Balance.PaidSoFar /
-        (info.Balance.RemainingFunds + info.Balance.PaidSoFar)
-    ) * 100
+  return Number(
+    (100n * info.Balance.PaidSoFar) /
```

Comment:
I think this means that the percentage value we return is "truncated"?
 IE: If there's `65.8941% paid` this function would return `65%`?

Reply:
I'm going to try out something on this branch which avoids the need for percentages altogether.